x-pack/legacy/plugins/console_extensions/spec/generated/ccr.forget_follower.json

```diff
@@ -6,6 +6,6 @@
     "patterns": [
       "{indices}/_ccr/forget_follower"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/ccr-post-forget-follower.html"
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/current"
```

Comment:
More https, please 🔒:)

Reply:
I'll open a PR upstream - these are generated and it looks like there's a few missing links too

Reply:
https://github.com/elastic/elasticsearch/pull/48479